### DISCUSSION TITLE
Change Oplitas Insurgency name

### DISCUSSION THREAD

**Discussion by Scrublord1013** [Vdauphin/HeartsAndMinds#1637] [Question and answer] [answered]
I was wondering if there was a way to change the insurgency name from Oplitas to something else? Im making one for SOG and want to make things as immersive as possible.

**Comment by STyx2909** [comment]
you have to edit the name into the files by yourself.

That's a feature I requested a long  time ago.

Le mer. 31 juil. 2024 à 08:49, Scrublord1013 ***@***.***> a
écrit :

> I was wondering if there was a way to change the insurgency name from
> Oplitas to something else? Im making one for SOG and want to make things as
> immersive as possible.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Vdauphin/HeartsAndMinds/discussions/1637>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASIGOOOH4UWRP7OGCMIKIT3ZPCCJNAVCNFSM6AAAAABLXZZFIGVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHE4TONBXGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
STyx2909

**Reply by Scrublord1013** [reply]
ive been trying to find what file its in but so far no luck

**Reply by Vdauphin** [reply, marked as answer]
Hello,

There are in =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml
https://github.com/search?q=repo%3AVdauphin%2FHeartsAndMinds%20Oplitas&type=code

Cheers

**Reply by Scrublord1013** [reply]
Thank you, of course its gonna be the one place i didnt look
